x-pack/plugins/enterprise_search/public/applications/app_search/components/schema/components/schema_callouts.tsx

```diff
@@ -129,3 +136,37 @@ export const ConfirmSchemaButton: React.FC = () => {
     </EuiButton>
   );
 };
+
+export const MissingSubfieldsCallout: React.FC = () => {
+  const { incompleteFields } = useValues(SchemaLogic);
+
+  return (
+    <EuiCallOut
+      iconType="alert"
+      color="warning"
+      title={`${incompleteFields.length} ${i18n.translate(
```

Comment:
suggestion: There's two issues with using translation this way. 

1) The `incompleteFields.length` value should be interpolated inside the translation string, using a value key. Most languages would not start this sentence with the number, so hardcoding the order like this will produce wonky sentences in translation.
2) Because incompleteFields.length can be `1`, we'll need to use a plural conditional to manage the pluralization. 

The resulting code should look something like this:

```
'title={i18n.translate(xpack.enterpriseSearch.appSearch.engine.schema.incompleteFields.title',
        { defaultMessage: '{count, number} {count, plural, one {field is} other {fields are}} missing subfields',
           values: { count: incompleteFields.length }
        }
      )}`
```

Reply:
aha! thank you!! is there documentation somewhere for `i18n`? with it being imported from `@kbn/i18n` it wasn't clear to me what we were using.

i'm happy to make these changes.